hardware/raspberrypi/mechanical/README.md

```diff
@@ -47,3 +47,17 @@ Also applicable to the A+.
 ## Raspberry Pi TV HAT
 
 - [Board](rpi_MECH_TVHAT_1p0.PNG)
+
```

Comment:
I know I linked to hardware/raspberrypi/mechanical/README.md in #1704 , but in hindsight I think this stuff belongs under hardware/computemodule/ rather than hardware/raspberrypi/ ?

Reply:
I think it's OK here. Just another mechanism for people to find the files they want. We do need to revamp this whole system for finding datasheets, but this should be good enough for the moment.